sling/nlp/parser/trainer/flow/builder.py

```diff
@@ -0,0 +1,248 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""Myelin function builder and expression evaluator."""
+
+import math
+from struct import pack
+
+from flow import Variable
+from flow import Function
+from flow import Flow
+
+DT_INT = "int32"
+DT_FLOAT = "float32"
+
+math_kernels = {
```

Comment:
I think we can delete this, since we are not using this currently.

Reply:
Done